### DISCUSSION TITLE
ETH-ALEP-ZIL on Windows

### DISCUSSION THREAD

**Discussion by pabloguns** [Lolliedieb/lolMiner-releases#1625] [General]
Hello! Does anyone have the .bat file to mine ETH +ALP +ZIl on Windows?

ETH 2miners

Zil Shardpool

**Comment by EFIMACHINE** [comment]
lolMiner.exe --algo ETHASH --pool eu.ezil.me:5555 --user ETH_WALLET.ZIL_WALLET --worker workername --dualmode ALEPHDUAL --dualpool fi.alephium.herominers.com:1199 --dualuser alephwallet --maxdualimpact 2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0 --enablezilcache

configure the maxdualimpact command yourself

**Reply by pabloguns** [reply]
Thank you!! I have this .bat with the "watchDog" included. Could you tell me if something is wrong, in some GPUs it gives me an error, is the overclock different in TONCOIN and in ALEPHIUM?

my .bat

@echo off

setlocal enableDelayedExpansion

Rem #################################
Rem ## Begin of user-editable part ##
Rem #################################

rem ALPH
set "ALEPHPOOL=fi.alephium.herominers.com:1199"
set "ALEPHWALLET="

rem ETH
set "POOLETH=us-eth.2miners.com:2020"
set "WALLETETH="

rem ZIL
set "POOLZIL=us1-zil.shardpool.io:3333"
set "WALLETZIL=

					
set "EXTRAPARAMETERS="

Rem #################################
Rem ##  End of user-editable part  ##
Rem #################################

if exist "%CD%\lolMiner.exe" goto infolder
echo "Searching for lolMiner.exe, because is not in this folder.That could take sometime..."
for /f "delims=" %%F in ('dir /b /s "C:\lolMiner.exe" 2^>nul') do set MyVariable=%%F
if exist "%MyVariable%" goto WindowsVer
echo "lolMiner.exe is not found in the system, that could be blocked by Windows Defender or Antivirus "
goto END

:infolder
set MyVariable=%CD%\lolMiner.exe

:WindowsVer
echo "Running lolMiner from %MyVariable%"
for /f "tokens=4-5 delims=. " %%i in ('ver') do set VERSION=%%i.%%j
if "%version%" == "10.0" goto W10
goto OtherW

:W10
"%MyVariable%" --algo ETHASH --pool !POOLZIL! --user !WALLETETH! --pass !WALLETZIL!@!POOLETH! --dualmode ALEPHDUAL --dualpool !ALEPHPOOL! --dualuser !ALEPHWALLET! --apiport 44444 --enablezilcache !EXTRAPARAMETERS! --watchdog exit
timeout 10
goto W10
goto END

:OtherW
"%MyVariable%" --algo ETHASH --pool !POOLZIL! --user !WALLETETH! --pass !WALLETZIL!@!POOLETH! --dualmode ALEPHDUAL --dualpool !ALEPHPOOL! --dualuser !ALEPHWALLET! --apiport 44444 --enablezilcache !EXTRAPARAMETERS! !EXTRAPARAMETERS! --watchdog exit --nocolor
timeout 10
goto OtherW

:END
pause
